modules/gapi/include/opencv2/gapi/garray.hpp

```diff
@@ -328,17 +329,21 @@ namespace detail
 template<typename T> class GArray
 {
 public:
+    // Host type (or Flat type) - the type this GArray is actually
+    // specified to.
+    using HT = typename detail::flatten_g<typename std::decay<T>::type>::type;
+
+    explicit GArray(const std::vector<HT>& v) // Constant value constructor
+        : m_ref(detail::GArrayU(detail::VectorRef(v))) { putDetails(); }
+    explicit GArray(std::vector<HT>&& v)      // Constant value move-constructor
```

Comment:
It is not constant value move constructor, it is just a move constructor

Reply:
I think constant value may be misinterpreted here and should be described in more details with mentioned context or just dropped

Reply:
Fixed (dropped).